lua/nvim-lsp-installer/servers/gopls.lua

```diff
@@ -0,0 +1,31 @@
+local server = require('nvim-lsp-installer.server')
+
+local root_dir = server.get_server_root_path('go')
+
+local install_cmd = [=[
+if ! command -v go &> /dev/null;
+then 
+  echo "Please install the Go CLI before installing gopls.";
+  echo "refer to https://golang.org/doc/install";
+  exit 1;
+fi
```

Comment:
I think you could move this to the `pre_install_check` function instead, something like:

```lua
return server.Server:new {
    ...,
    pre_install_check = function ()
        if vim.fn.executable("go") ~= 1 then
            error("Please install the Go CLI before installing gopls (https://golang.org/doc/install).")
        end
    end
}
```

Reply:
Nice! Much much cleaner! Adding it right away. 
-> added with https://github.com/ecmma/nvim-lsp-installer/commit/5eb4be95b86f3c558eef0b815ec8041fba60f917